system/Language/en/Publisher.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+// Publisher language settings
+return [
+	'expectedFile'      => 'Publisher::{0} expects a valid file.',
+	'expectedDirectory' => 'Publisher::{0} expects a valid directory.',
+	'collision'         => 'Publisher encountered an unexpected {0} while copying {1} to {2}.',
+];
```

Comment:
@MGatner it seems the new files are unrelated to the PR?

Reply:
Wow yeah sorry about that! I think I forgot to `-u` my stash so the new files carried over. I will force back to the correct file.